Src/Plugins/Security/Security.csproj

```diff
@@ -28,7 +28,7 @@
     <PackageReference Include="AWSSDK.IdentityManagement" Version="3.5.0.61" />
     <PackageReference Include="GoogleApi" Version="4.0.4" />
     <PackageReference Include="Microsoft.Security.Utilities" Version="1.1.0" />
-    <PackageReference Include="MongoDB.Driver" Version="2.13.1" />
+    <PackageReference Include="MongoDB.Driver" Version="2.15.0" />
```

Comment:
>2.15.0 [](http://example.com/codeflow?start=56&length=6)

Version 2.15.0 contains the updated version of SharpCompress (0.31.0), which does not have a vulnerability.